x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -192,46 +286,84 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
         />
       );
     },
+    getDefaultDescription: ({ field, policy_name: policyName, target_field: targetField }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.enrich', {
+        defaultMessage:
+          'Enriches data to "{target_field}" if the "{policy_name}" policy matches "{field}"',
+        values: {
+          field,
+          policy_name: policyName,
+          target_field: targetField,
+        },
+      }),
   },
   fail: {
     FieldsComponent: Fail,
     docLinkPath: '/fail-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.fail', {
       defaultMessage: 'Fail',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.fail', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.fail', {
       defaultMessage:
         'Returns a custom error message on failure. Often used to notify requesters of required conditions.',
     }),
+    getDefaultDescription: ({ if: value }) =>
+      value
+        ? i18n.translate('xpack.ingestPipelines.processors.defaultDescription.fail', {
```

Comment:
```suggestion
        i18n.translate('xpack.ingestPipelines.processors.defaultDescription.fail', {
```

Reply:
The description for Fail wasn't rendering, and I think the errant `?` might be why.

Reply:
This `? :` combo is intentional at the moment. If we have no `if` value; I recall we said that we don't actually want to display these in the default descriptions anyway, might this work:

"Raises an exception that halts execution"